### ISSUE TITLE
Deprecation notices

### ISSUE THREAD

**Issue by strarsis** [andyjansson/postcss-sassy-mixins#4]
When compiling with postcss-sassy-mixins added as a processor:

```
Container#eachAtRule is deprecated. Use Container#walkAtRules instead.
Node#removeSelf is deprecated. Use Node#remove.
```

**Comment by EduardoRT** [comment]
I think it's fixed now, but the version number wasn't bumped so the update doesn't reach us? 

`https://github.com/andyjansson/postcss-sassy-mixins/commit/3c9d430cda27b6f7788ce0a7a35ae61d4ce0adc1`

**Comment by andyjansson** [comment]
Fixed in v2.0.
